zerver/models.py

```diff
@@ -4458,3 +4458,16 @@ def is_authenticated(self) -> bool:
         request.user.is_authenticated verifications.
         """
         return True
+
+
+class StreamTopic(models.Model):
```

Comment:
I'm going to change this to just Topic, at Greg's suggestion.

Reply:
sgtm